frontend/express/public/core/events/javascripts/countly.overview.models.js

```diff
@@ -165,7 +304,24 @@
                 topEvents: [],
                 monitorEvents: {},
                 monitorEventsData: [],
-                selectedDatePeriod: "30days"
+                selectedDatePeriod: "30days",
+                configureEventsList: [],
+                overviewGroupData: [],
+                configureOverview: [],
+                eventMapping: {},
+                eventProperties: [{
+                    "label": CV.i18n("events.overview.count"),
```

Comment:
why are label and value same here and coming from localizations?
so if you are checking based on value in your code and if the localization/language changes the check will fail.

Reply:
It is being handled inside getEventProperties. But anyways the value initialization is wrong there, I'll correct that :)